### ISSUE TITLE
Distribute pre-built noms command

### ISSUE THREAD

**Issue by aboodman** [attic-labs/noms#1694]
Once #1693 is done, it would be really easy to distribute a pre-built noms command, rather than requiring devs build from source.

Go includes a cross-compiler, so as part of our continuous build we could just build for whatever platforms we want to support, and post those things on a website someplace.

If there's just one command, installing noms is as easy as downloading the binary and putting it in your path.

**Comment by mikegray** [comment]
Duplicate of #2306
